src/System.Net.Http/ref/project.json

```diff
@@ -3,14 +3,15 @@
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Net.Primitives": "4.0.0",
+    "System.Net.Primitives": "4.0.10",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
Yes System.Net.Http is part of standard library. The easiest way to think about standard library is to see what is in NETStandard.Library (https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets) meta-package which we are representation we are using for standard library. I know today the implementation depends on Crypto but now after this change the public API surface will as well. At any rate I chatted with @KrzysztofCwalina offline and I think we are OK adding Crypto to standard library.

Can you please add System.Security.Cryptography.X509Certificates to the list at https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets?